.gitlab-ci.yml

```diff
@@ -36,3 +41,43 @@ release:
   variables:
     RELEASE_PY_VERSION: python3.7
     GIT_DEPTH: 0
+  after_script:
+    - pip install "setuptools_scm>=6.2,<8" "setuptools-git-versioning<2"
+    - python -m setuptools_git_versioning | tee VERSION.txt
+  artifacts:
+    paths:
+      - VERSION.txt
+    expire_in: 1 week
+
+trigger:
+  stage: .trigger
+  dependencies:
+    - release
+  parallel:
+    matrix:
+      - TRIGGERED_REPO: 29
+        NAME: inspector
+      - TRIGGERED_REPO: 7
+        NAME: gdcapi
+      - TRIGGERED_REPO: 122
+        NAME: sheepdog
+      - TRIGGERED_REPO: 79
+        NAME: esbuild
+      - TRIGGERED_REPO: 30
+        NAME: graphmanager
+      - TRIGGERED_REPO: 76
+        NAME: gdcadmin
```

Comment:
What are the numbers?

Reply:
arbitrary project_id from gitlab

Reply:
👍🏻 👍🏻 And I take they won't change afterwards - no matter what.